spark3/src/test/java/org/apache/iceberg/spark/source/TestPathIdentifier.java

```diff
@@ -59,7 +59,7 @@
   @Before
   public void before() throws IOException {
     tableLocation = temp.newFolder();
-    identifier = new PathIdentifier(tableLocation.getAbsolutePath());
+    identifier = new PathIdentifier(tableLocation.getAbsolutePath(), null, null);
```

Comment:
Rather than changing this file, can you add a constructor to fill these in as defaults?

Reply:
Ok